model/src/main/java/com/epam/esm/model/entity/Entity.java

```diff
@@ -0,0 +1,4 @@
+package com.epam.esm.model.entity;
+
+public abstract class Entity {
```

Comment:
should implement Serializable instead of its childs

Reply:
done